checks/default.nix

```
@@ -0,0 +1,15 @@
+{ inputs, ... }: {
+  perSystem = { pkgs, ... }: {
+    checks = {
+      tofu-validate = pkgs.runCommand "tofu-validate" {
+        nativeBuildInputs = [ pkgs.opentofu ];
+      } ''
+        set -e
+        tofu validate
+        mkdir -p $out
+        echo "Validation OK" > $out/success
```

Not a strong opinion but I usually I see people doing directly this in these cases:
```suggestion
  touch $out
```
If you want you can print stuff directly to the stdout, logs will be collected by nix as usual (`nix log`).